crates/miden-testing/src/tx_context/context.rs

```diff
@@ -58,13 +58,12 @@ impl TransactionContext {
         code: &str,
         assembler: Assembler,
     ) -> Result<Process, ExecutionError> {
-        let (stack_inputs, mut advice_inputs) = TransactionKernel::prepare_inputs(
+        let (stack_inputs, advice_inputs) = TransactionKernel::prepare_inputs(
             &self.tx_inputs,
             &self.tx_args,
             Some(self.advice_inputs.clone()),
         )
         .unwrap();
-        advice_inputs.extend(self.advice_inputs.clone());
```

Comment:
Is it correct that previously we were extending the advice inputs with the `self.advice_inputs` from the Context twice? First time in the `prepare_inputs` procedure (`let mut advice_inputs = init_advice_inputs.unwrap_or_default();`) and the second time here? If so, it is a great catch! The code becomes more clear.